adev/BUILD.bazel

```
@@ -157,6 +157,7 @@ architect(
         "--poll=1000",
         "--live-reload",
         "--watch",
+        "--verbose",
```

This is the only way I found to log the build issues. 
Without this, when the build fails, it fails silently.